src/panels/config/automation/trigger/types/ha-automation-trigger-geo_location.ts

```diff
@@ -23,87 +22,63 @@ export class HaGeolocationTrigger extends LitElement {
     };
   }
 
+  protected firstUpdated(): void {
+    if (!this.hass) {
+      return;
+    }
+
+    this._schema = [
```

Comment:
Generate schema using memoizeOne that takes `localize: LocalizeFunc`. That way when translations update, the schema updates too.